tools/util_linux.go

```diff
@@ -48,15 +49,19 @@ func CheckCloneFileSupported(dir string) (supported bool, err error) {
 }
 
 func CloneFile(writer io.Writer, reader io.Reader) (bool, error) {
-	fdst, fdstFound := writer.(*os.File)
-	fsrc, fsrcFound := reader.(*os.File)
-	if fdstFound && fsrcFound {
-		if _, _, err := syscall.Syscall(syscall.SYS_IOCTL, fdst.Fd(), BtrfsIocClone, fsrc.Fd()); err != 0 {
```

Comment:
It looks like the old and new code are intended to be equivalent, but that's hard to see from the diff.  To make it easier for future readers trying to reason about this code, could we limit the refactoring here and simply change the constant `BtrfsIocClone` to `ioctlFiClone`?

Reply:
Good suggesting. Sorry for refactoring contamination. Let' me do that.